staging/src/k8s.io/component-helpers/storage/volume/pv_helpers_test.go

```diff
@@ -146,23 +149,28 @@ func TestFindMatchVolumeWithNode(t *testing.T) {
 		}),
 		makeTestVolume("affinity-pv", "affinity001", "100G", true, func(pv *v1.PersistentVolume) {
 			pv.Spec.StorageClassName = "wait"
+			pv.Spec.VolumeAttributesClassName = &classGold
```

Comment:
added.

Reply:
I think it would be better if we kept all the existing test cases, and then added new ones with VAC. It looks like right now we have only kept one of the original test cases?

Reply:
@msau42 updated.